bookOrders/models.py

```diff
@@ -1,16 +0,0 @@
-from django.db import models
-
-from books.models import Book
-from supplier.models import Supplier
-
-
-# Create your models here.
-class bookOrder(models.Model):
```

Comment:
First character of the class name should be capital
Add class meta for the admin

Reply:
Also add relevant helping text + error messages